### PR TITLE
Use custom @PublicApi instead of @Beta #7450

### PR DESCRIPTION
**Pull request by rymsha**

### CODE REVIEW

**Comment by GlennRicaud** [comment]
Approved. I let you fix the conflicts and merge

File: modules/core/core-api/src/main/java/com/enonic/xp/audit/AuditLogIds.java [hunk 1]
```diff
@@ -7,12 +7,15 @@
 import java.util.stream.Collectors;
 
 import com.google.common.annotations.Beta;
+import com.google.common.base.Function;
```
Comment (jsi) [inline modules/core/core-api/src/main/java/com/enonic/xp/audit/AuditLogIds.java]: Codacy Issue found: [Unused import - com.google.common.base.Function.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=4731894)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=4731894&bid=15798613)

File: modules/core/core-api/src/main/java/com/enonic/xp/audit/AuditLogIds.java [hunk 2]
```diff
@@ -7,12 +7,15 @@
...
+import com.google.common.collect.Collections2;
```
Comment (jsi) [inline modules/core/core-api/src/main/java/com/enonic/xp/audit/AuditLogIds.java]: Codacy Issue found: [Unused import - com.google.common.collect.Collections2.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=4731894)

**Pull request merged**